flink-yarn-tests/src/test/java/org/apache/flink/yarn/YARNITCase.java

```diff
@@ -78,6 +83,9 @@ public void testPerJobMode() {
 		jobGraph.addJar(new org.apache.flink.core.fs.Path(testingJar.toURI()));
 
 		YarnClusterClient clusterClient = yarnClusterDescriptorV2.deployJobCluster(clusterSpecification, jobGraph);
+
+		clusterClient.shutdown();
+		yarnClusterDescriptorV2.close();
```

Comment:
Closing in finally or try-with-resource not needed?

Reply:
True will refactor it.